crates/types/src/config/aws.rs

```diff
@@ -31,4 +32,19 @@ pub struct AwsOptions {
     /// https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html
     /// Can be overridden by the `AWS_EXTERNAL_ID` environment variable.
     pub aws_assume_role_external_id: Option<String>,
+
+    /// # Request Compression threshold
+    ///
+    /// Request minimum size to enable compression. Default: 4MB
```

Comment:
Maybe leave a comment here to say why 4 and not 6

Also, is 4mb of b64 data plus the other stuff like headers and the curly braces and quotes definitely always less than 6mb?  Wondering if the limit should be defined in terms of total lambda payload and we calculate ourself based on the headers etc whether we need to compress